cli/smartcontract/testdata/rpcbindings/royalty/rpcbindings.out

```diff
@@ -0,0 +1,58 @@
+// Code generated by neo-go contract generate-rpcwrapper --manifest <file.json> --out <file.go> [--hash <hash>] [--config <config>]; DO NOT EDIT.
+
+// Package royalty contains RPC wrappers for Test royalty contract.
+package royalty
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	_ "github.com/nspcc-dev/neo-go/pkg/rpcclient/nep24"
```

Comment:
Good, we see that import is present, and no code for notification is emitted. But this import it's unused because `nep24` is actually not used by the auto-generated code. Hence I'd suggest not to add this package in case if only NEP24Payable is supported by the contract.